lib/spack/docs/workflows.rst

```diff
@@ -0,0 +1,1150 @@
+=========
+Workflows
+=========
+
+The process of using Spack involves building packages, running
+binaries from those packages, and developing software that depends on
+those packages.  For example, one might use Spack to build the
+`netcdf` package, use `spack load` to run the `ncdump` binary, and
```

Comment:
These should probably be double backticks. Single backticks just make the text italic.